content/blog/nats-security-update.md

```diff
@@ -0,0 +1,68 @@
++++
+categories = ["General"]
+date = "2019-03-19"
+tags = ["security", "c", "cncf", "cure53", "gnatsd", "go", "node"]
+title = "NATS.io Security Update"
+author = "Colin Sullivan"
++++
+
+Often times, blogs or articles with titles like this one will contain bad
+news.  That isn’t the case here!  We’re sharing the results of our
+security review, and how that bolsters our new security features in the
+upcoming NATS server 2.0 release, expected in Q2 2019.
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-2">
+    <div class="thumbnail">
+      <img class="img-responsive center-block" alt="NATS Security Image" src="/img/blog/nats-security-update/nats-security.png">
+    </div>
+  </div>
+</div>
+
+## Security Audit
+
+In November of last year, the CNCF sponsored [cure53](https://cure53.de/)
+to perform a security audit of the NATS server and some of the more
+popular NATS clients.  We setup a secure server for cure53 to attack
+and they analyzed our source code.
+
+_“Cure53 chose a two-pronged approach to testing against NATS. In order to
+maximize coverage, the testers performed a source code audit, as well as
+engaged in classic penetration testing against a NATS-provided cloud
+instance.”_
+
+Emphasis was on the [NATS server](https://github.com/nats-io/gnatsd) and our
+canonical client, the [go client](https://github.com/nats-io/go-nats),
+since it is used as a reference for most other maintainer supported
+clients.  The [C client](https://github.com/nats-io/cnats) and [Node.js](https://github.com/nats-io/node-nats) client were also tested.  
+
+## Summary
+
+Overall, we feel NATS came out with flying colors.  The server had one
+low rated vulnerability which was immediately fixed.
+
+Simplicity pays off.  The report noted that most clients are small in
```

Comment:
Simplicity pays off. The report noted that most of the NATS clients are small in